### DISCUSSION TITLE
BLtouch Not completing full 25 points.

### DISCUSSION THREAD

**Discussion by TheZenYeti** [Jyers/Marlin#627] [Q&A] [answered]
Hey thank you to this great community for guiding me through my first 3D printing experience. I have an issue with my BLtouch where it will run through 20 leveling points and then half probe the 21st spot. I get a message saying that ABL is finished and the print will begin. on larger prints it becomes an issue when the back of the bed isnt actually level. running E3V2-BLtouch-10x10 v4.2.2. Im stumped on how to fix this and could use some help. Thank you.

**Comment by TheZenYeti** [comment]
edit 5x5 sorry

**Comment by Jyers** [comment, marked as answer]
Sounds like it could be a case of probe failure. There is a resistor mod you can do to rectify it, or you can try probing with the heaters off.

**Reply by TheZenYeti** [reply]
Heaters off offers no change, I've checked wiring. I thought the resistor issue was only for older boards? I cant find anything on google about modding the resistor. Thanks for the reply

**Reply by Jyers** [reply]
Here's a post with the mod I'm referring to. https://www.facebook.com/groups/ender3v2/permalink/811429246136763/

**Reply by TheZenYeti** [reply]
THANK YOU. Jyers you do so much for us mate. thank you thank you thank you.

**Reply by Jyers** [reply]
Happy to help! Definitely let me know how it goes and if that ends up solving your issue :)
